en/orm/query-builder.rst

```diff
@@ -299,6 +299,15 @@ portable::
     $query = $articles->find();
     $query->select(['count' => $query->func()->count('*')]);
 
+Note that most of the functions accept an additional argument to specify the types to bind to the arguments and/or the return type,
+for example:
+
+
+    $query->select(['minDate' => $query->func()->min('date', ['text']);
+
+
+For details, see the documentation for [FunctionsBuilder](https://api.cakephp.org/4.1/class-Cake.Database.FunctionsBuilder.html)
```

Comment:
```suggestion
For details, see the documentation for `FunctionsBuilder <https://api.cakephp.org/4.1/class-Cake.Database.FunctionsBuilder.html>`__
```

Reply:
I wonder if we can setup a redirect for `api.cakephp.org/4/class-` urls to the latest release.

Reply:
Derp I should be using the 4.x redirect.